pkgs/development/python-modules/djhtml/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "djhtml";
+  version = "3.0.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-84/U59V1U4snkcXKorVMU5etwI+PjSZYU+Ha+oojmuk=";
+  };
+
```

Comment:
There is a [test suite](https://github.com/rtts/djhtml/tree/main/tests). We should run the tests. It could require to get the source from GitHub (`fetchFromGitHub`) instead of PyPI.